src/main/java/core/basesyntax/StatisticField.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum StatisticField {
```

Comment:
Better not use enum because enum values are always capital letters. Create constants instead